src/App.tsx

```diff
@@ -1,14 +1,66 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from './app/hooks';
+import { actions } from './features/todos';
+import { Todo } from './types/Todo';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const dispatch = useDispatch();
+  const { todos } = useAppSelector(state => state.todos);
+  const [showLoader, setShowLoader] = useState(false);
```

Comment:
Let's give a better name for the boolean state `snowLoader -> isShowLoader`

Reply:
done